core/src/test/java/org/elasticsearch/common/lucene/search/MultiPhrasePrefixQueryTests.java

```diff
@@ -76,8 +78,11 @@ public void testBoost() throws Exception {
         IndexReader reader = DirectoryReader.open(writer, true);
         MultiPhrasePrefixQuery multiPhrasePrefixQuery = new MultiPhrasePrefixQuery();
         multiPhrasePrefixQuery.add(new Term[]{new Term("field", "aaa"), new Term("field", "bb")});
+        //nocommit: seems like this test still makes sense? or shall we rather forbid usage of setBoost already?
```

Comment:
the test makes sense indeed. +1 to ban Query.setBoost in src/main but not src/test like @nik9000 suggests. I think this should work if you only touch dev-tools/src/main/resources/forbidden/core-signatures.txt instead of all-signatures.txt.